_posts/2020-06-17-ombulabs-open-source-guidelines.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "OmbuLabs Open Source Guidelines"
+date: 2020-06-17 11:16:00
+categories: ["open-source", "guideline"]
+author: bronzdoc
+---
+
+![goHbayOl3PPzk7yYgEEOzF8q9zqRurIwZexM1ge8.jpg](https://tettra-production.s3.us-west-2.amazonaws.com/0d6efb4f154041e899af17bdcd19c1b5/ce8d5f452da02a88554d80c256162a35/d822b155a4112474fdb7aea5ee22465e/d28647d6b4d6e5842508679707ee21a7/goHbayOl3PPzk7yYgEEOzF8q9zqRurIwZexM1ge8.jpg)
+
+At OmbuLabs we dedicate some of our work hours to open source every week. Also some of us contribute to open source during our spare time.
+
+Here are a few tips to help us keep our OSS contributions organized and encourage people to contribute
+
+<!--more-->
+
+From now on every-time we contribute to an OSS project in working hours, we fork it to the [FastRuby.io organization](https://github.com/fastruby).
+
+This way we can keep track of the contributions we made as a company, also the branches created in the project are accessible to the rest of the team.
+
+As a company we get these benefits:
+
+\- We use OSS projects and we want them to be better
```

Comment:
I'm not sure this belong here, is this one of the reasons we fork to FastRuby?

Reply:
Yes